### PR TITLE
Jetpack Cloud: Add new Backups Upsell, refactor common components from Scan Upsell

### PR DESCRIPTION
**Pull request by cbauerman**
#### Changes proposed in this Pull Request

* Add jetpack cloud upset component based on Scan Upsell
* Use new upsell component in scan and backup upsells

screenshot

#### Testing instructions

1. navigate to `/backups` for a site without backups
2. verify the backup works and looks as expected
3. repeat with `/scan`

### CODE REVIEW

File: client/landing/jetpack-cloud/sections/backups/controller.js [hunk 1]
```diff
@@ -19,6 +23,20 @@ export function wrapInSiteOffsetProvider( context, next ) {
...
 
+export function showUpsellIfNoBackup( context, next ) {
+	context.primary = (
+		<UpsellSwitch
+			UpsellComponent={ BackupsUpsell }
+			display={ context.primary }
+			getStateForSite={ getRewindState }
+			QueryComponent={ QueryRewindState }
+		>
+			<p>{ '...' }</p>
```
Comment (ChaosExAnima) [inline client/landing/jetpack-cloud/sections/backups/controller.js:34]: Is this intentional?

Comment (cbauerman) [reply client/landing/jetpack-cloud/sections/backups/controller.js:34]: just a leftover from when I copied the scan example and didn't have a backups placeholder to put there. I will get rid of it 👍

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/3447206

Thank you @cbauerman for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/3447206).

**Pull request merged**